Rakefile

```diff
@@ -4,3 +4,45 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+namespace :unicorn do
+  desc "Start unicorn for development env."
+  task(:start) {
+    config = rails_root + "config/unicorn.rb"
+    sh "bundle exec unicorn_rails -c #{config} -E production -D -p 8080"
+  }
+
+  desc "Stop unicorn"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)